apps/api/src/dto/form-field.dto.ts

```diff
@@ -8,4 +8,13 @@ export const CreateFormFieldSchema = z.object({
 	type: z.enum(FormInput),
 });
 
+export const EditFormFieldSchema = z.object({
+	label: z.string().optional(),
+	placeholder: z.string().optional(),
+	require: z.boolean().optional(),
+	type: z.enum(FormInput).optional(),
```

Comment:
type is not optional

Reply:
for updating schema

Reply:
We can disable type conversion for now, because, If we change the types after some form submission, it will break in UI when rendering the responses